packages/contracts/README.md

```diff
@@ -46,107 +48,105 @@ cd contracts
 ```
 
 Install `npm` packages:
+
 ```shell
 yarn install
 ```
 
 ### Running Tests
+
 Tests are executed via `yarn`:
+
 ```shell
 yarn test
 ```
 
 Run specific tests by giving a path to the file you want to run:
+
 ```shell
 yarn test ./test/path/to/my/test.spec.ts
 ```
 
 ### Measuring test coverage:
+
 ```shell
 yarn test:coverage
 ```
 
 The output is most easily viewable by opening the html file in your browser:
+
 ```shell
 open ./coverage/index.html
```

Comment:
`open` is a mac specific thing, one particular flavor of linux uses `xdg-open`

Reply:
I'll fix this in another PR